### ISSUE TITLE
Add Keyframes like in MMD?

### ISSUE THREAD

**Issue by ghost** [ShinHogera/CM3D2.Maidirector.Plugin#1]
Sugesstion: Would be better if it just saved the current maids possition as a keyframe? So you could actually do it frame by frame or something.

**Comment by ShinHogera** [comment]
Yes, that's a planned feature. I'll try implementing it once I finish saving/loading.
